impl/src/main/java/org/glassfish/soteria/mechanisms/jaspic/HttpBridgeServerAuthModule.java

```diff
@@ -73,14 +77,33 @@
  */
 public class HttpBridgeServerAuthModule implements ServerAuthModule {
 
+    private static final Logger LOGGER =  Logger.getLogger(HttpBridgeServerAuthModule.class.getName());
+
         private CallbackHandler handler;
         private final Class<?>[] supportedMessageTypes = new Class[] { HttpServletRequest.class, HttpServletResponse.class };
         private final CDIPerRequestInitializer cdiPerRequestInitializer;
-        
+        private final Map<String, Class<?>> mappings = new HashMap<>();
+
         public HttpBridgeServerAuthModule(CDIPerRequestInitializer cdiPerRequestInitializer) {
             this.cdiPerRequestInitializer = cdiPerRequestInitializer;
+            try {
+                initMappings();
+            } catch (ClassNotFoundException e) {
+                LOGGER.warning(e.getMessage());
+            }
         }
-        
+
+    private void initMappings() throws ClassNotFoundException {
+        mappings.put("Basic", BasicAuthenticationMechanism.class);
+        mappings.put("Form", FormAuthenticationMechanism.class);
+        mappings.put("CustomForm", CustomFormAuthenticationMechanism.class);
+        mappings.put("JWT", Thread.currentThread().getContextClassLoader().loadClass("fish.payara.microprofile.jwtauth.eesecurity.JWTAuthenticationMechanism"));
+        mappings.put("Certificate", Thread.currentThread().getContextClassLoader().loadClass("fish.payara.security.realm.mechanisms.CertificateAuthenticationMechanism"));
+        mappings.put("Azure", Thread.currentThread().getContextClassLoader().loadClass("fish.payara.security.openid.azure.AzureOpenIdAuthenticationMechanism"));
+        mappings.put("Google", Thread.currentThread().getContextClassLoader().loadClass("fish.payara.security.openid.google.GoogleOpenIdAuthenticationMechanism"));
+        mappings.put("OAuth2", Thread.currentThread().getContextClassLoader().loadClass("fish.payara.security.oauth2.OAuth2AuthenticationMechanism"));
```

Comment:
**Missing OpenIdAuthenticationMechanism:**
mappings.put("OIDC", Thread.currentThread().getContextClassLoader().loadClass("fish.payara.security.openid.OpenIdAuthenticationMechanism"));

Reply:
Also missing the `fish.payara.security.authentication.twoIdentityStore.TwoIdentityStoreAuthenticationMechanism`